app/src/main/java/com/mobiledevelopmentfoodapp/MainActivity.kt

```diff
@@ -37,23 +37,23 @@ class MainActivity : ComponentActivity() {
                         modifier = Modifier.fillMaxWidth(),
                         color = MaterialTheme.colors.background
                     ) {
-                        Header("Android")
+                        header("Android")
                     }
-                    ButtonBar()
-                    Menu()
+                    buttonBar()
+                    menu()
```

Comment:
changed function names to begin with lower case letter to adhere to https://kotlinlang.org/docs/coding-conventions.html#function-names

Reply:
As much as I want to agree with you here (and do agree with you here)... actually... I can't.
In Compose, function names should be nouns, and should be PascalCase (UpperCamelCase).  So, best to leave this as is.

https://android.googlesource.com/platform/frameworks/support/+/androidx-main/compose/docs/compose-api-guidelines.md